index.js

```diff
@@ -232,7 +267,11 @@ webix.ui(
                     {
                         view: "multiview", gravity: 5,
                         cells: [
-                            {id: "Dashboard", cols: [tableInMainPart, formInMainPart]},
+                            {id: "Dashboard", cols: [
+                                {rows: [tabbarForFilteringYear,tableInMainPart]},
+                                    {width: 400, rows:[formInMainPart]}
```

Comment:
unnecessary rows layout for form 
and please check code formatting for this part